_pages/cfp.md

```diff
@@ -1,12 +1,145 @@
+
+Acceptances/Rejections Sent: Ooctober 31 2017
+
+## Selection Process
+
+Here is how our selection process works: 
+
+1. When you submit your proposal, your name and contact information will be anonymized.
+2. Once the CFP closes, proposals are distributed among the members of our CFP Review Committee.
+3. The review process has two stages:
+	
+	a) The CFP Review Committee members review their assigned proposals individually, and give the proposal a score. 
+
+	b) The CFP Review Committee gathers to discuss the proposals and to make a list of potential accepted talks.
+4. Speaker names and contact info are de-anonymized and there is a period for the CFP Review Committee to confirm their choices of proposals to accept.
+5. All applicants are informed if their talk was accepted or rejected.
+
+## Guidelines 
+
+We ask for three things other than your basic info: An abstract, a timeline, and a short answer question. Please keep in mind that these are things you must cover in your proposal, but are not limited to.
+
+**Your submission must follow our [Code of Conduct](/coc/).** 
+
+**Do not include any personally identifying information in your abstract, timeline or answer to the short answer question. This includes:**
+- **Your name.**
+- **Where you work/go to school.**
+- **Any relation to the StarCon organizers or members of the CFP Review Committee.**
+- **Your gender, ethnicity, sexual orientation, religion (or lack of) etc.**
+
+### Abstract 
+
+A brief summary of your talk. The reviewers will use this to determine the relevance of the topic and to get a feel for what the talk will cover. 
+
+Your abstract should: 
+- Be approximately 200 words. 
+- Include:
+	- The title of your talk. 
+	- What the audience can expect to learn from your talk. 
+ 	- Why you think the topic of the talk is interesting, important etc. 
+
+Example: 
+
+_Maintaining good relationships with your co-workers is vital to both personal and organizational development. In my talk Stepping on Keyboards for Fun and Profit I will cover the basics of the technique and how it can improve relationships with your peers, or your human roommates. Stepping on Keyboards (SoK) is a relatively unknown team-building technique among humans, however it is very well known among the feline community. SoK prioritizes getting in the way of a human while they are sitting at their computer, preferably typing, and demanding that they pay attention to you, thus improving communication skills among any human-cat team. The audience will be introduced to the topic, learn several of the common techniques and see a live demo. I believe that this technique is an untapped resource for most teams and that it has the potential to have a large impact on team-building and team development for small to large scale organizations._
+
+### Timeline 
+
+The point of this is to get a feel for what your talk will be like. Unlike the abstract, this will not be posted on the website if your talk is selected. 
```

Comment:
I think applicants would find it confusing that it says that reviewers will use the abstract to get a "feel for what the talk will cover" and then later says that we'll using the timeline to "get a feel for what your talk will be like". Say something like `The purpose of this is to get an idea of the composition of your talk`?